03.process-wiki-output.py

```diff
@@ -132,21 +107,22 @@ def process_nationality(nationality):
 
 namefile = open(args.names, 'r')
 outfile = open(args.outfile, 'w')
-outfile.write("id"+args.sep+"name_first"+args.sep+"name_middle"+args.sep+"name_last"+args.sep+"ethnicity"+args.sep+"country\n")
+outfile.write("id"+args.sep+"name"+args.sep+"ethnicity"+args.sep+"country\n")
```

Comment:
For future reference, best to use the builtin csv module here. https://docs.python.org/3/library/csv.html